src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+     * @type {number}
+     */
+    maxFeatures: PropTypes.number,
+    /**
+     * Geometry name to use in a BBOX filter.
+     * @type {string}
+     */
+    geometryName: PropTypes.string,
+    /**
+     * Optional list of property names to serialize.
+     * @type {String[]}
+     */
+    propertyNames: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * Filter condition. See http://openlayers.org/en/latest/apidoc/ol.format.filter.html
+     * for more information.
+     * @type {object}
+     */
+    filter: PropTypes.object,
+    /**
+     * The ol.map where the map will zoom to.
+     *
+     * @type {Object}
+     */
+    map: PropTypes.instanceOf(OlMap).isRequired,
+    /**
+     * The minimal amount of characters entered in the input to start a search.
+     * @type {Number}
+     */
+    minChars: PropTypes.number,
+    /**
+     * A render function which gets called with the selected item as it is
+     * returned by the server. It must return an `AutoComplete.Option` with
+     * `key={feature.id}`.
+     * The default will display the property `name` if existing or the `id` to.
+     *
+     * @type {function}
+     */
+    renderOption: PropTypes.func,
+    /**
+     * An onSelect function which gets called with the selected feature as it is
+     * returned by server.
+     * The default function will create a searchlayer, adds the feature and will
+     * zoom to its extend.
+     * @type {function}
+     */
+    onSelect: PropTypes.func,
+    /**
+     * Options which are added to the fetch-POST-request. credentials is set to
+     * 'same-origin' as default but can be overwritten.
+     * @type {object}
+     */
+    additionalFetchOptions: PropTypes.object
+  }
+
+  static defaultProps = {
+    srsName: 'EPSG:3857',
+    outputFormat: 'application/json',
+    minChars: 3,
+    additionalFetchOptions: {},
+    /**
+     * Create an AutoComplete.Option from the given data.
+     *
+     * @param {Object} feature The feature as returned by the server.
+     * @return {AutoComplete.Option} The AutoComplete.Option that will be
+     * rendered for each feature.
+     */
+    renderOption: feature => {
+      return (
+        <Option key={feature.id}>
+          {feature.properties.name ? feature.properties.name : feature.id}
+        </Option>
+      );
+    },
+    /**
+     * The default onSelect method if no onSelect prop is given. It zooms to the
+     * selected item.
+     *
+     * @param {object} feature The selected feature as returned by the server.
+     * @param {ol.map} olMap The openlayers map that was passed via prop.
+     */
+    onSelect: (feature, olMap) => {
+      if (feature) {
+        const olView = olMap.getView();
+        const geoJsonFormat = new OlFormatGeoJSON();
+        const olFeature = geoJsonFormat.readFeature(feature);
+        const geometry = olFeature.getGeometry();
+
+        if (geometry) {
+          olView.fit(geometry, {
+            duration: 500
+          });
```

Comment:
These options might be a good candidate for configuration, right?

Reply:
As it's just in the default of the `onSelect` property this doesn't need to be configurable. You can just provide a complete new function if you want a different duration.